lib/checkunusedfunctions.cpp

```diff
@@ -231,6 +231,53 @@ void CheckUnusedFunctions::parseTokens(const Tokenizer &tokenizer, const char Fi
 }
 
 
+static bool isOperatorFunction(const std::string & funcName) {
+    /* Operator functions are invalid function names for C, so no need to check
+     * this in here. As result the returned error function might be incorrect.
+     *
+     * List of valid operators can be found at:
+     * http://en.cppreference.com/w/cpp/language/operators
+     *
+     * Conversion functions must be a member function (at least for gcc), so no
+     * need to cover them for unused functions.
+     *
+     * To speed up the comparision, not the whole list of operators is used.
+     * Instead only the character after the operator prefix is checked to be a
+     * none alpa numeric value, but the '_', to cover function names like
+     * "operator_unused". In addition the following valid operators are checked:
+     * - new
+     * - new[]
+     * - delete
+     * - delete[]
+     */
+    const std::string operatorPrefix = "operator";
+    if (funcName.compare(0, operatorPrefix.length(), operatorPrefix) != 0) {
+        return false;
+    }
+
+    const char firstOperatorChar = funcName[operatorPrefix.length()];
+    if (firstOperatorChar == '_') {
+        return false;
+    }
+
+    if (!std::isalnum(firstOperatorChar)) {
+        return true;
+    }
+
+    const std::vector<std::string> additionalOperators = {
```

Comment:
MSVC 10 doesn't like this, see https://ci.appveyor.com/project/danmar/cppcheck/build/1.73.5252/job/9n9j7jw7tlyhag4l#L63
Maybe use `make_container` (https://github.com/danmar/cppcheck/blob/master/lib/utils.h#L31)?